### PR TITLE
Updated the cloud portal article with the new dashboard

### PR DESCRIPTION
**Pull request by jonat123**
Should first be merged when the new dashboard has been fully released.

ETA this or next week.

### CODE REVIEW

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md [hunk 1]
```diff
@@ -11,17 +11,31 @@ The Umbraco Cloud Portal helps you manage your Umbraco Cloud project. From here,
...
-For quick access, you can *favorite* the projects you are currently working on, to make them appear as the first projects in the view. You can also browse through the projects by using the Search field.
+### Project Groups
+
+To give a better overview over your projects it is possible to order your projects into **Groups**.
+
+This can be done by clicking the **Edit Group** button in the top left side.
+
+![Edit Groups](images/edit-group.png)
+
+When clicking on **Edit Groups** you can create new Groups to sort your project in and create a better overview for yourself.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md:34]:
```suggestion
After clicking on **Edit Groups**, you can create new Groups to sort your project in and create a better overview for yourself.
```

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md [hunk 2]
```diff
@@ -11,17 +11,31 @@ The Umbraco Cloud Portal helps you manage your Umbraco Cloud project. From here,
...
+
+![Create Group](images/add-group.png)
+
+Once you click **Add Group** you can give it a name that fit the group, and then you can drag and drop your projects into the group of your choice.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md:38]:
```suggestion
Click **Add Group** to give it a name and then drag and drop your projects into the group of your choice.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good, @jonat123 🙌 Just a couple of suggestions.

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md [hunk 3]
```diff
@@ -11,17 +11,31 @@ The Umbraco Cloud Portal helps you manage your Umbraco Cloud project. From here,
 
 When you log in to the [Umbraco Cloud Portal](https://www.s1.umbraco.io/home/login/) you will be presented with a view of all the Umbraco Cloud projects you've created or have been added to as a team member.
 
-![View all projects](images/view-projects-v9.png)
+![View all projects](images/Umbraco-cloud-dashboard.png)
 
-By default, the projects are ordered by the latest updates - projects you've worked on recently will be shown on top and the ones you haven't worked on for a while are at the bottom. In the top-right corner of the Cloud Portal, you will find:
+On the project dashboard, you get an overview of all your projects, the environments added, the usage for each of the project, which plan it is on, and whether it is a baseline or baseline-child project.
```
Comment (jonat123) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md:16]:
```suggestion
On the project dashboard, you get an overview of all your projects, the environments, the usage for each project, which plans it is on, and whether it is a baseline or baseline-child project.
```

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md [hunk 4]
```diff
@@ -11,17 +11,31 @@ The Umbraco Cloud Portal helps you manage your Umbraco Cloud project. From here,
 
 When you log in to the [Umbraco Cloud Portal](https://www.s1.umbraco.io/home/login/) you will be presented with a view of all the Umbraco Cloud projects you've created or have been added to as a team member.
 
-![View all projects](images/view-projects-v9.png)
+![View all projects](images/Umbraco-cloud-dashboard.png)
 
-By default, the projects are ordered by the latest updates - projects you've worked on recently will be shown on top and the ones you haven't worked on for a while are at the bottom. In the top-right corner of the Cloud Portal, you will find:
+On the project dashboard, you get an overview of all your projects, the environments, the usage for each project, which plans it is on, and whether it is a baseline or baseline-child project.
```
Comment (jonat123) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md:16]:
```suggestion
The projects dashboard gives an overview of all your projects, their environments, usage for each project, which plans it's on, and whether it's a baseline or baseline-child project.
```

File: Umbraco-Heartcore/Getting-Started-Cloud/The-Cloud-Portal/index.md [hunk 5]
```diff
@@ -10,9 +10,9 @@ In this article, you will learn more about the Umbraco Cloud Portal and what opt
 
 After you have logged into the [Umbraco Cloud Portal](https://www.umbraco.io), you are presented with the project list overview.
 
-![Portal Project Overview](images/portalOverview.png)
+![Portal Project Overview](images/Heartcore-dashboard.png)
 
-From here you can see a list of all your projects. If you have several projects, you can search for specific projects by using the search bar in the top right corner.
+From here you can see a list of all your projects. If you have several projects, you can sort specific projects in groups.
```
Comment (jonat123) [inline Umbraco-Heartcore/Getting-Started-Cloud/The-Cloud-Portal/index.md:15]:
```suggestion
From here you can see a list of all your projects. If you have multiple projects, you can sort specific projects into groups.
```

**Pull request merged**